server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -0,0 +1,137 @@
+use crate::{number::next_number, requisition::common::get_lines_for_requisition};
```

Comment:
The processor diagrams describe the logic here and show examples of how records `travel` around and when transfers are create/updated